5.1.3/httpd.conf

```diff
@@ -2,13 +2,13 @@
 # This is the main Apache HTTP server configuration file.  It contains the
 # configuration directives that give the server its instructions.
 # See <URL:http://httpd.apache.org/docs/2.2> for detailed information.
-# In particular, see
+# In particular, see 
```

Comment:
Let's not change `httpd.conf` as it'd break legacy apps that are based on this image. 

If anyone needs a change on this, they can mount in their own (or extend).